test/unit/helpers/form_helper_test.rb

```diff
@@ -0,0 +1,78 @@
+require "test_helper"
+
+class FormHelperTest < ActionView::TestCase
+  context "form_errors" do
+    should "return an unordered list with errors when there are errors present" do
+      expected = "<ul class=\"help-block error-block\"><li>One</li><li>Two</li><li>Three</li></ul>"
+
+      assert_equal expected, form_errors(%w[One Two Three])
+    end
+
+    should "return an empty unordered list when there are no errors present" do
+      expected = "<ul class=\"help-block error-block\"></ul>"
+
+      assert_equal expected, form_errors
+    end
+  end
+
+  context "form_group" do
+    setup do
+      object = stub(field_name: "", errors: Hash.new([]))
```

Comment:
It'd be worth explaining this cryptic `Hash.new([])` that we came up with in the commit message.